File: C1908GLeThanhNghi/AP/25-11-2020/myApp/myApp/Models/Rectangle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace myApp.Models
{
    public class Rectangle
    {
        private float length;
        private float width;
        public float Length {
            get {
                return length;
            }
            set {
                length = (value < 0.0f || value > 20.0f) ? 0.0f : value;
                //length = value;
            }
        }
        public float Width { get; set; }
    }
}

File: C2408G/Semester2/JP1/myapp/src/main/java/com/aptech/myapp/models/Rectangle.java
package com.aptech.myapp.models;

public class Rectangle extends Shape{
    protected float width;
    protected float height;

    @Override
    public double getArea() { //phương thức abstract bắt buộc lớp con phải overide
        return width * height;
    }


    @Override
    public String toString() { //Ko bắt buộc vì toString ko phải là abstract method
        System.out.println(super.name);//vì name là protected
        //System.out.println(this.color)//ko dc vì color ko phải public/protected;
        System.out.println(super.getColor());
        return super.toString();
    }

}
